### DISCUSSION TITLE
OSPO Training

### DISCUSSION THREAD

**Discussion by fbim-genosr** [todogroup/ospology#193] [OSPO Discussions]
Hi everyone,

can someone recommend a training on how to start an OSPO in governmental context? I've found something a bit more general at linuxfoundation https://training.linuxfoundation.org/training/open-source-management-and-strategy/

Cheers

**Comment by debbryant** [comment]
So far as I know, there are no formalized trainings for starting an OSPO in the public sector. Your best bet is to review existing best practices for a few that have them, there are examples in the EU, in the US you can find the Federal Consumer Protections Bureau’s open source first policy (and as I recall other operational info that supported policy)  published on GitHub.  For the most part (in my experience) they’re adapting OSPO known practices in the public sector to their agencies own operating environment, and look to their peers for use case.  But as the EU is investing in OSPOs now, you might check there to see if they’ve created any recently.  I can send you some contacts off list of you’d like.

There are materials published by the TODO Group, Red Hat and others on this list general practices for OSPO.

Deb Bryant

**Comment by caniszczyk** [comment]
We have a set of OSPO101 modules https://github.com/todogroup/ospo101 - but not all of it is going to be applicable to governments - it's something we are working on though on adding a module OSPO101 cc: @anajsana

**Reply by anajsana** [reply]
@fbim-genosr Since OSPO 101 is seen as a starting point to get started into OSPOs, we could add a specific module on Governments to the existing course as @caniszczyk suggested (if there is nothing out there yet). I would love to see other communities and organizations contributing to making the course more diverse!

FYI, I'm working with other people from the community to create more advanced courses as a continuation of the OSPO 101 modules (Like a course to contribute and launch open source projects, as a continuation of [module 7](https://github.com/todogroup/ospo101/tree/main/module7#open-source-project-creation-overview) and based on the [Outbound OSS Guide](https://todogroup.org/guides/outbound-oss/) content released ).

**Reply by anajsana** [reply]
@fbim-genosr maybe you could add this need for training for OSPOs in a governmental context as a new comment in the [OSPO Wishlist](https://github.com/todogroup/ospology/discussions/174)? 🙂

**Comment by ashleywolf** [comment]
There's the OSPO++ network too that might have some pointers.

> OSPO++ is a network and a community of collaborative open source program offices in universities, governments, and civic institutions. We're building resources to help create OSPOs, actively engaging in discussions on how to best manage and grow open source programs, and how to garden sustainable communities that last.

https://ospoplusplus.com/

**Comment by will-ww** [comment]
Maybe useful: https://github.com/X-lab2017/open-wonderland/discussions/127

- OSPOs! New Approaches to Public Sector Open Source
- The OSPO – A New Tool for Digital Government

**Comment by fbim-genosr** [comment]
Thank you all very much for your help!
